cg/services/illumina_services/illumina_post_processing_service/illumina_post_processing_service.py

```diff
@@ -77,7 +84,9 @@ def store_illumina_sample_sequencing_metrics(
             sample_metrics_dto=sample_metrics, sequencing_run=sequencing_run
         )
 
-    def store_illumina_flow_cell_data(self, flow_cell_dir_data: IlluminaRunDirectoryData) -> None:
+    def store_illumina_data_in_status_db(
+        self, flow_cell_dir_data: IlluminaRunDirectoryData
```

Comment:
```suggestion
        self, sequencing_run: IlluminaRunDirectoryData
```
Might as well do this now.

Reply:
Or is this a `demultiplexing_run`?

Reply:
but sequencing run is already used for the database object, which should be which?

Reply:
fixed